src/recording-page.ts

```diff
@@ -1,8 +1,25 @@
+interface Point {
```

Comment:
is this point interface different than the one in the types.ts? Either rename them differently or remove one of them.

Reply:
I removed this one